examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/ConfigureFragment.kt

```diff
@@ -5,45 +5,66 @@ import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.lifecycle.lifecycleScope
 import androidx.navigation.fragment.findNavController
 import com.google.android.material.dialog.MaterialAlertDialogBuilder
 import com.revenuecat.purchases.Purchases
 import com.revenuecat.purchases.PurchasesConfiguration
 import com.revenuecat.purchases.amazon.AmazonConfiguration
 import com.revenuecat.purchases_sample.R
 import com.revenuecat.purchases_sample.databinding.FragmentConfigureBinding
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.flow.onEach
+import kotlinx.coroutines.launch
 import java.net.MalformedURLException
 import java.net.URL
 
 class ConfigureFragment : Fragment() {
 
     lateinit var binding: FragmentConfigureBinding
 
+    private lateinit var dataStoreUtils: DataStoreUtils
+
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View {
+        dataStoreUtils = DataStoreUtils(requireActivity().applicationContext.configurationDataStore)
```

Comment:
We don't have any injection framework on this app, so I kind of hacked this together... There might be better ways but this works for now.